index.js

```diff
@@ -0,0 +1,68 @@
+const { v4: uuidv4 } = require('uuid');
+
+async function getDomainUrl({ zipModifier }) {
```

Comment:
I think it's better to pass `zipModifier` as the first argument (in `addImages` as well). Since it's not really part of the options, it is required for this function to work.